cli-feed-v4.json

```diff
@@ -46,7 +46,7 @@
       "hidden": false
     },
     "v4-prerelease": {
-      "release": "4.26.1",
+      "release": "4.27.0",
```

Comment:
@v-smanchem I see that we are missing the `endOfLifeDate` property which was added in #395 .We need to carry it to all new pre-release versions.
cc @cloudmelon

Reply:
@v-smanchem Could you please fix this in the latest version?